### DISCUSSION TITLE
Potential error in examples related to free surface generation

### DISCUSSION THREAD

**Discussion by IGNaval-git** [capytaine/capytaine#408] [Q&A] [answered]
Hi,

I have been trying to run the examples, some work flawlessly, others do not seem to respond appropriately. 
The main example is the "animate_free_surface", in which when I run it reaches a "ValueError" in the file "list_of_points.py". Trace back can be found at the end of the post.

I have debugged it and I think that I traced up to _normalize_points in the module "lists_of_points" (as per the traceback log), in line 49 where points.ndim == 2. The free surface, passed to that function as the variable "points", does not have an ndim attribute, which causes it to crash.

I would like to know if someone is experiencing the same issues and maybe if I am running something wrong or I am missing some dependencies.

I installed everything with pip.

========= Traceback 

INFO:   New floating body: sphere_0.
INFO:   Clipping sphere_0 with respect to Plane(normal=[0. 0. 1.], point=[0. 0. 0.])
INFO:   Solve DiffractionProblem(body=sphere_0, omega=2.000, water_depth=inf, wave_direction=0.000).
INFO:   Solve RadiationProblem(body=sphere_0, omega=2.000, water_depth=inf, radiating_dof='Heave').
Traceback (most recent call last):
  File "N:\Repos\capytaine-master\docs\user_manual\examples\animate_free_surface.py", line 31, in <module>
    diffraction_elevation_at_faces = solver.compute_free_surface_elevation(free_surface, diffraction_result)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\capytaine\bem\solver.py", line 272, in compute_free_surface_elevation
    points, output_shape = _normalize_free_surface_points(points, keep_mesh=True)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\capytaine\tools\lists_of_points.py", line 36, in _normalize_free_surface_points
    points, output_shape = _normalize_points(points, keep_mesh)
  File "C:\Users\User\AppData\Local\Programs\Python\Python310\lib\site-packages\capytaine\tools\lists_of_points.py", line 28, in _normalize_points
    raise ValueError("This should not happen.")
ValueError: This should not happen.

**Comment by mancellin** [comment, marked as answer]
Thank you for your feedback.

I believe this is the same as #392.

**Reply by IGNaval-git** [reply]
I understand, thank you for your reply.
